### PR TITLE
Set up testing environment for Client

### PR DESCRIPTION
**Pull request by johnnysee**
Link to PT: https://www.pivotaltracker.com/story/show/179900591

## Tasks

- Add `Cypress` for acceptance testing
- Add `start-server-and-test` development dependency
- Add `cypress` script with `start-server-and-test` for running `Cypress`
- Set `baseUrl` in `cypress.json`
- Disable `chromeWebSecurity` for Cypress browser
- Run `yarn cypress` and delete generated tests

### CODE REVIEW

**Review by Fraseer — COMMENTED** [review]
Looks good to me

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:3000",
```
Comment (dorianbuck) [inline cypress.json:2]: small thing, but if we make client side run on :3001. we can avoid issues with running backend simultaneously.

**Review by tochman — APPROVED** [review]
Good job

File: cypress.json (hunk 1 above)
Comment (maxarvid) [reply cypress.json:2]: A valid point! Will keep as 3000 for now.

**Review by AugustEnhager — COMMENTED** [review]
LGTM!

**Pull request merged**